### PR TITLE
Consolidate information on Spark into a single cookbook

### PR DESCRIPTION
**Pull request by wlach**
Information on Spark was spread between the (soon-to-be-gone) "tools"
section and the BigQuery access document. Consolidate this into
a single cookbook under BigQuery (since 99% of the material is really
about using Spark with BigQuery).

### CODE REVIEW

File: src/tools/spark.md [hunk 1]
```diff
@@ -102,18 +117,20 @@ core_pings_single_day = spark.read.format("bigquery") \
     .select("client_id", "experiments", "normalized_channel")
 ```
 
+You can see this code in action in [this example Python notebook on DataBricks](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
```
Comment (akkomar) [inline src/tools/spark.md:120]:
```suggestion
You can see this code in action in [this example Python notebook on Databricks](https://dbc-caf9527b-e073.cloud.databricks.com/#notebook/141939).
```
^typo

File: src/tools/spark.md [hunk 2]
```diff
@@ -123,6 +140,7 @@ and is typically suitable for reading smaller amounts of data. If you need
 to read large amounts of data, prefer the Storage API as described above.
 
 Example:
+
 ```python
```
Comment (akkomar) [inline src/tools/spark.md:145]:
```suggestion
```
Remove empty line

File: src/tools/spark.md [hunk 3]
```diff
@@ -151,8 +169,7 @@ rows = query_job.result().to_dataframe()
```
Comment (akkomar) [inline src/tools/spark.md:169]:
```suggestion
```
Remove empty line

File: src/tools/spark.md (hunk 2 above)
Comment (wlach) [reply src/tools/spark.md:145]: I think the extra line helps with readability (when viewing it as markdown), so am inclined to keep it.

Comment (akkomar) [reply src/tools/spark.md:145]: This way it's not consistent with other code blocks in the same file.

Comment (wlach) [reply src/tools/spark.md:145]: Ah ok, I'll do a quick followup

**Pull request merged**